front/.gitignore

```diff
@@ -32,3 +32,5 @@ yarn-error.log*
 *.njsproj
 *.sln
 .vscode
+graphql.schema.json
+
```

Comment:
Since both `graphql.schema.json` and `schema.ts` are deleted, shouldn't we add both to the gitignore ?